dashboard/src/components/myProgress/discover/DiscoverProjectsPage.vue

```diff
@@ -160,6 +170,9 @@ aria-label="search for projects to pin"></b-input>
             </div>
           </template>
         </b-table>
+        <div>
+          <contact-owners-dialog v-if="contactModal.show && isEmailEnabled"  :projectName="contactModal.projectName"
```

Comment:
please move `contact-owners-dialog` declaration to the bottom of the most outer `div`, looks like that would go after the `div` on the line 200